broker/alert_acquisition/ztf.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""Retrieve and parse alerts from the ZTF."""
+
+import os
+
+import pandas as pd
+
+from .._utils import setup_log
```

Comment:
I would just called it `utils.py`.  I don't think the underscore in the filename to denote some sort of "protected" or "non-public" functions is that important here.